doc-source/quick-start.rst

```diff
@@ -5,124 +5,160 @@
+    $ compliance --fetch --evidence local -C auditree_demo.json
 
-* Run the checks::
+  Running this command generates a Git repository locally at ``/$TMPDIR/compliance``
+  and it executes all fetchers found within the ``demo_example`` package.
 
-    $ compliance --check 'accred1,accred2' .
+* Running the checks::
 
-  This executes the checks that are associated with the specified accreditations,
-  creates the reports, executes notifiers (you can see the messages in the output),
-  and generates a file called ``check_results.json`` with detailed information about
-  the execution.  The file is put into the evidence locker by default.
-  Accreditation to check mapping is handled in the ``controls.json`` file.  More
-  on that below.
+    $ compliance --check demo.arboretum.accred,demo.custom.accred --evidence local -C auditree_demo.json
+
+  Running this command executes checks associated with the specified accreditations,
+  creates check result reports in the evidence locker, executes notifiers (you can
+  see the messages in the output), updates the evidence locker README with a check
+  report table of contents and generates a file in the evidence locker root called
+  ``check_results.json`` with detailed information about the check execution.
 
 Advanced Topics
 ---------------
 
+Configuration
+~~~~~~~~~~~~~
+
+There are two configuration files used to configure the execution of the
+``auditree-framework``.  The main configuration file can be named anything you
+like.  In the case of our demo we've chosen the name ``auditree-demo.json``.  There
+is also a check to accreditation mapping configuration file and this file must be
+named ``controls.json``.
+
+controls.json
+~~~~~~~~~~~~~
+
+The mapping between a check (check class path) and accreditations
+(a list of accreditations) happens in the ``controls.json`` configuration
+file.  Each new check must be included in ``controls.json`` in
+order to be considered for execution by the Auditree framework.
+Note that this is not the case for fetchers, where all are executed.
+As an example, the format for ``controls.json`` is as follows::
+
+  {
+    "chk_pkg.chk_cat_foo.checks.chk_module_foo.FooCheckClass": ["accred.one"],
+    "chk_pkg.chk_cat_bar.checks.chk_module_bar.BarCheckClass": ["accred.one", "accred.two"]
+  }
 
-Fetchers
-~~~~~~~~
+Remote Evidence Locker
+~~~~~~~~~~~~~~~~~~~~~~
 
-You might want to push the evidences generated by ``--fetch`` into a
-remote git repository. For that, you can use ``--evidence`` and ``-C
-setup.json``. This ``setup.json`` contains a fake URL of the repo, you
-can edit it to match the URL you want. However, you have to provide
-GitHub credentials. By default, you should use
-``~/.credentials`` (or use ``--creds-path`` to point somewhere else)::
+Storing and using evidence in a Git remote hosting service repository is achievable
+by using the ``--evidence`` option with settings of either ``no-push`` or
```

Comment:
Make it clear that no push only retrieves

Reply:
done in https://github.com/ComplianceAsCode/auditree-framework/pull/77/commits/cb7696375a7fdbe40a96a450056725ed0dc40852